src/databaseaccessor/accessor.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ACCESSOR_H
+#define ACCESSOR_H
+#include "connectiongetter.h"
+
+class Accessor
+{
+public:
+    Accessor();
+    ~Accessor()= default;
+protected:
+    void setQuery(QSqlDatabase db);
+    void execQuery(QString query);
+    QSqlQuery query;
+private:
+    Connection *database;
```

Comment:
Please don't forget about this rule:
https://en.wikipedia.org/wiki/Rule_of_three_(C%2B%2B_programming)